mix.exs

```diff
@@ -81,7 +81,7 @@ defmodule Uptime.MixProject do
   # See the documentation for `Mix` for more info on aliases.
   defp aliases do
     [
-      setup: ["deps.get", "ecto.setup", "assets.setup", "assets.build"],
+      setup: ["deps.get", "ecto.setup", "cmd npm install --prefix assets", "assets.setup", "assets.build"],
```

Comment:
did this change not get merged in yet, maybe need a rebase?

Reply:
must have been merge issue mismanaged